c++/test/TestWriter.cc

```diff
@@ -699,6 +704,98 @@ namespace orc {
     }
   }
 
+#ifndef _MSC_VER
```

Comment:
leave some comments to explain why you skip windows here.

Reply:
Done.